website/content/docs/concepts/seal.mdx

```diff
@@ -8,145 +8,130 @@ description: >-
 
-Once a Vault node is unsealed, it remains unsealed until one of these things happens:
+Once you unseal a Vault node, it remains unsealed until one of the following happens:
 
-1. It is resealed via the API (see below).
+1. You reseal it using the API.
+1. You restart the server.
+1. Vault's storage layer encounters an unrecoverable error.
 
-2. The server is restarted.
-
-3. Vault's storage layer encounters an unrecoverable error.
-
--> **Note:** Unsealing makes the process of automating a Vault install
+Unsealing nodes can make automating a Vault installation
 difficult. Automated tools can easily install, configure, and start Vault,
-but unsealing it using Shamir is a very manual process. For most users
-Auto Unseal will provide a better experience.
+but unsealing it using Shamir seals is a manual process. For most users,
+auto unseal provides a better experience.
 
 ## Sealing
 
-There is also an API to seal the Vault. This will throw away the root
-key in memory and require another unseal process to restore it. Sealing
-only requires a single operator with root privileges.
+There is also an API to seal the Vault. The API deletes the root
+key from memory and requires another unseal process to restore it. Sealing the Vault requires only a single operator with root privileges.
 
-This way, if there is a detected intrusion, the Vault data can be locked
-quickly to try to minimize damages. It can't be accessed again without
-access to the root key shares.
+If there is a detected intrusion, you can seal the Vault data quickly to try to minimize damages. Users cannot access the data from a sealed Vault without access to the root key shares.
 
 ## Auto unseal
 
-Auto unseal was developed to aid in reducing the operational complexity of
+Auto unseal aids in reducing the operational complexity of
 keeping the unseal key secure. This feature delegates the responsibility of
-securing the unseal key from users to a trusted device or service. At startup
-Vault will connect to the device or service implementing the seal and ask it
-to decrypt the root key Vault read from storage.
+securing the unseal key from users to a trusted device or service. At startup,
+Vault connects to the trusted device or service and prompts it
+to decrypt the root key from storage.
 
 ![Auto Unseal](/img/vault-auto-unseal.png)
 
-There are certain operations in Vault besides unsealing that
-require a quorum of users to perform, e.g. generating a root token. When
-using a Shamir seal the unseal keys must be provided to authorize these
-operations. When using Auto Unseal these operations require _recovery
-keys_ instead.
+There are certain Vault operations besides unsealing that
+require a quorum of users to perform, for example, generating a root token. When
+you use a Shamir seal, you must provide the unseal keys to authorize these
+operations. When you use auto unseal, these operations require recovery
+keys instead.
```

Comment:
Rewrites for conciseness, removes Latin phrase "e.g.", rewrites passive voice, makes feature name lowercase, and removes italics for emphasis.